src/contexts/useWhoamiContext.js

```diff
@@ -0,0 +1,15 @@
+import React, { createContext, useContext, useState } from 'react'
```

Comment:
afaict we're not using this so please remove this file.

Reply:
Done.  This was spill-over; I ended up extracting this work into a different branch, but didn't double-check the cleanup.